netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -64,8 +64,10 @@
 @CanIgnoreReturnValue
 public final class NettyChannelBuilder
     extends AbstractManagedChannelImplBuilder<NettyChannelBuilder> {
-
-  public static final int DEFAULT_FLOW_CONTROL_WINDOW = 1048576; // 1MiB
+  /**
+   * 1MiB.
+   */
+  public static final int DEFAULT_FLOW_CONTROL_WINDOW = 1024 * 1024;
```

Comment:
This comment format is meant for javadocs, while the text doesn't fit into javadocs. The original format seems more appropriate for an explanatory comment.

ditto for the other places.

Reply:
Thanks for your suggestions, i got it. @zhangkun83